adr/cli.py

```diff
@@ -96,10 +104,10 @@ def _build_parser_arguments(parser, config):
     # positional arguments
     parser.add_argument('task', nargs='*')
     parser.add_argument(
-        '-h',
-        '--help',
-        action='store_true',
+        '-h', '--help', action='store_true',
         help="Type --help to get help.\nType <recipe> --help to get help with a recipe.")
+    parser.add_argument('-of', '--output-file', type=str,
```

Comment:
I don't think you can use multi-character options with a single `-`. Let's use `-o`

Reply:
The code ran well with "-of", but I changed to "-o" as you suggested. "-o" looks more consistent with other arguments.